agent/util-scripts/gold/pbench-register-tool/test-46.txt

```diff
@@ -19,7 +19,6 @@ Available tools:
 	blktrace
 	bpftrace
 	cpuacct
-	dcgm
```

Comment:
This actually made me think: should we sort the list of keys so dcgm and node-exporter don't get moved to the end of the list?

Reply:
@dbutenhof, just do confirm in writing, do you think we can do that later, or would you want to see the sorting done in this PR?

Reply:
@portante @Maxusmusti I approved the PR, which I figured would be clear enough; but for the record, yes, sure. It's a pretty trivial UX issue; it'd be more consistent and maybe a bit more clear, but I doubt most people really pay attention to the full list of tools in the usage.

Reply:
I can easily change that once Chuck's work has been integrated, I'll be sure to sort in the future